assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/components/table/styles.scss

```diff
@@ -1,89 +1,127 @@
-			tr.ee-rspnsv-table-body-row {
-				@include max782px {
-					height: auto;
-					width: 100%;
-				}
+	tbody.ee-rspnsv-table-body {
+		tr.ee-rspnsv-table-body-row {
+			@include max782px {
+				height: auto;
+				width: 100%;
+			}
 
-				.ee-rspnsv-table-body-td {
-					&:not(.ee-rspnsv-table-hide-on-mobile) {
-						@include max782px {
-							display: flex;
-							padding: var(--ee-padding-micro);
-						}
+			.ee-rspnsv-table-body-td {
+				&:not(.ee-rspnsv-table-hide-on-mobile) {
+					@include max782px {
+						display: flex;
+						padding: var(--ee-padding-micro);
 					}
 				}
 			}
 		}
+	}
+
+	th,
+	td {
+		padding: var(--ee-padding-tiny) var(--ee-padding-micro);
+	}
 
-		th,
-		td {
-			padding: var(--ee-padding-tiny) var(--ee-padding-micro);
+	input,
+	select {
+		width: 100%;
+	}
+
+	&__actions {
+		&.ee-rspnsv-table-header-th,
+		&.ee-rspnsv-table-body-td,
+		&.ee-rspnsv-table-footer-th {
+			min-width: 85px;
+			width: 85px;
 		}
 
-		input,
-		select {
-			width: 100%;
+		.ee-icon-button {
+			margin-bottom: 0;
 		}
+	}
 
-		&__actions {
-			&.ee-rspnsv-table-header-th,
-			&.ee-rspnsv-table-body-td,
-			&.ee-rspnsv-table-footer-th {
-				min-width: 85px;
-				width: 85px;
-			}
+	&__price-id {
+		&.ee-rspnsv-table-header-th,
+		&.ee-rspnsv-table-body-td,
+		&.ee-rspnsv-table-footer-th {
+			min-width: 40px;
+			width: 60px;
+		}
+	}
 
-			.ee-icon-button {
-				margin-bottom: 0;
-			}
+	&__price-type {
+		&.ee-rspnsv-table-header-th,
+		&.ee-rspnsv-table-body-td,
+		&.ee-rspnsv-table-footer-th {
+			min-width: 120px;
+			width: 120px;
 		}
+	}
 
-		&__price-id {
-			&.ee-rspnsv-table-header-th,
-			&.ee-rspnsv-table-body-td,
-			&.ee-rspnsv-table-footer-th {
-				min-width: 40px;
-				width: 60px;
-			}
+	&__amount {
+		&.ee-rspnsv-table-header-th,
+		&.ee-rspnsv-table-body-td,
+		&.ee-rspnsv-table-footer-th {
+			min-width: 120px;
+			width: 160px;
 		}
 
-		&__price-type {
-			&.ee-rspnsv-table-header-th,
-			&.ee-rspnsv-table-body-td,
-			&.ee-rspnsv-table-footer-th {
-				min-width: 120px;
-				width: 120px;
-			}
+		&.ee-rspnsv-table-header-th.ee-rspnsv-table-header-th.ee-number-column {
+			text-align: center;
 		}
+	}
 
-		&__amount {
-			&.ee-rspnsv-table-header-th,
-			&.ee-rspnsv-table-body-td,
-			&.ee-rspnsv-table-footer-th {
-				min-width: 120px;
-				width: 160px;
-			}
+	&__total .ee-input-base.ee-input-base.ee-input:disabled {
+		background: var(--ee-color-grey-14);
+		border-color: var(--ee-color-grey-9);
+	}
 
-			&.ee-rspnsv-table-header-th.ee-rspnsv-table-header-th.ee-number-column {
-				text-align: center;
-			}
-		}
+	.ee-rspnsv-table-footer {
+		@include max782px {
+			.ee-rspnsv-table-footer-row {
+				position: relative;
+				width: 100%;
+				height: 60px;
 
-		&__total .ee-input-base.ee-input-base.ee-input:disabled {
-			background: var(--ee-color-grey-14);
-			border-color: var(--ee-color-grey-9);
+				.ee-col-0,
+				.ee-col-1,
+				.ee-col-2 {
+					display: none;
+					padding: 0;
```

Comment:
if this is display: none, padding shouldn't be an issue

Reply:
hmmm... bet ya my browser was caching the previous page, then because I added the padding rule after the display none, because the padding was still affecting the layout